src/elm/Data/Currency.elm

```diff
@@ -19,6 +20,12 @@ map f (Currency c) =
     Currency <| f c
 
 
+(>=$) : Currency -> Currency -> Bool
+(>=$) (Currency c1) (Currency c2) =
+    c1 >= c2
+infix 4 >=$
```

Comment:
Thoughts on having a named function for this instead of an operator? http://package.elm-lang.org/help/design-guidelines#avoid-infix-operators

Perhaps something like `Currency.greaterThan` or `Currency.isMoreThan`?

In my experience, named functions are easier to read and easier to partially apply in pipelines and compositions.